crates/task/src/task_template.rs

```diff
@@ -189,7 +189,7 @@ impl TaskTemplate {
                     },
                 ),
                 command,
-                args: self.args.clone(),
+                args: args_with_substitutions,
```

Comment:
Previously, we were using the unsubstituted args. I'm not sure if this was intentional or a bug, I have changed it to substituted args here.